### DISCUSSION TITLE
Is there a way to push the whole memos to cloudflare pages with seach function / robots.txt??

### DISCUSSION THREAD

**Discussion by etjec4** [usememos/memos#2831] [Ideas]
I just want share knowledge without buying vps. : )
if anyone have ideas, please tell me.😘

**Comment by lincolnthalles** [comment]
Cloudflare Pages is intended for static websites. It can be used for documentation and knowledge bases, but it requires different [tools](https://developers.cloudflare.com/pages/framework-guides/).

Cloudflare Workers is also not suited for Memos. It requires a proper VPS or a Docker container hosting service.
